### ISSUE TITLE
Extend `SMW::SQLStore::AddCustomFixedPropertyTables` to allow for a custom prefix

### ISSUE THREAD

**Issue by mwjames** [SemanticMediaWiki/SemanticMediaWiki#1689]
Relational:

- https://github.com/SemanticMediaWiki/SemanticExtraSpecialProperties/issues/48#issuecomment-228091436
- https://github.com/SemanticMediaWiki/SemanticExtraSpecialProperties/pull/62

**Comment by mwjames** [comment]
```php
@@ -50,18 +50,21 @@ class PropertyTableDefinitionBuilder {
 	public function doBuild() {
 
 		$this->addTableDefinitionForDiTypes( $this->diTypes );
 
 		$this->addTableDefinitionForFixedProperties(
+			$this->fixedPropertyTablePrefix,
 			$this->specialProperties
 		);
 
 		$customFixedProperties = array();
+		$fixedPropertyTablePrefix = $this->fixedPropertyTablePrefix;
 
-		Hooks::run( 'SMW::SQLStore::AddCustomFixedPropertyTables', array( &$customFixedProperties ) );
+		Hooks::run( 'SMW::SQLStore::AddCustomFixedPropertyTables', array( &$customFixedProperties, &$fixedPropertyTablePrefix ) );
 
 		$this->addTableDefinitionForFixedProperties(
+			$fixedPropertyTablePrefix,
 			$customFixedProperties
 		);
 
 		$this->addRedirectTableDefinition();
 
@@ -145,20 +148,20 @@ class PropertyTableDefinitionBuilder {
 	}
 
 	/**
 	 * @param array $properties
 	 */
-	private function addTableDefinitionForFixedProperties( array $properties ) {
+	private function addTableDefinitionForFixedProperties( $fixedPropertyTablePrefix, array $properties ) {
 		foreach( $properties as $propertyKey => $propetyTableSuffix ) {
 
 			// Either as plain index array containing the property key or as associated
 			// array with property key => tableSuffix
 			$propertyKey = is_int( $propertyKey ) ? $propetyTableSuffix : $propertyKey;
 
 			$this->addPropertyTable(
 				DataTypeRegistry::getInstance()->getDataItemId( DIProperty::getPredefinedPropertyTypeId( $propertyKey ) ),
-				$this->fixedPropertyTablePrefix . strtolower( $propetyTableSuffix ),
+				$fixedPropertyTablePrefix . strtolower( $propetyTableSuffix ),
 				$propertyKey
 			);
 		}
 	}
 ```

**Comment by mwjames** [comment]
Has been addressed with #1890.
